decrypt_otpauth.py

```diff
@@ -116,38 +124,6 @@ def decode_object(self, index):
         self.unpacked_uids[index] = obj
         return obj
 
-# @click.command()
-# @click.option('--encrypted-otpauth-account',
-#               help="path to your encrypted OTP Auth account (.otpauth)",
-#               required=True,
-#               type=click.File('rb'))
-# def main(encrypted_otpauth_account):
```

Comment:
I'd rather keep this block because it documents how to open `.otpauth` files.

Reply:
Fixed in https://github.com/CooperRS/decrypt-otpauth-files/pull/1/commits/42bb25916aa19dcf8abe3700e5636851db47e3f9